src/main/java/com/hms_networks/americas/sc/extensions/historicaldata/HistoricalDataConstants.java

```diff
@@ -52,4 +54,16 @@
 
   /** Name of historical data queue time tracker file 2. */
   static final String QUEUE_TIME_FILE_2_NAME = "histDataTime2";
+
+  /** Minimum amount of Historical FIFO queue that can be set by library users. */
+  static final long MIN_QUEUE_SPAN_MINS = 1;
+
+  /** Default maximum amount of time Historical FIFO can get behind. */
+  static final long MAX_HIST_FIFO_GET_BEHIND_MS = SCTimeUnit.HOURS.toMillis(12);
+
+  /** Default thread sleep time waiting for EBD in ms. */
+  static final int DEFAULT_EBD_THREAD_SLEEP_MS = 5;
+
+  /** Maximum amount of time that EBD call should wait in milliseconds before timing out. */
+  static final long MAX_EBD_WAIT_MILLIS = 15000;
```

Comment:
To be consist ant with the other constants. 

```suggestion
  static final long MAX_EBD_WAIT_MS = 15000;
```

Reply:
changed.